### DISCUSSION TITLE
Add Custom field in release notes

### DISCUSSION THREAD

**Discussion by brux88** [rfennell/AzurePipelines#1240] [General]
hi, in my tasks I have added a custom "Cliente" field as I can add it next to the task. i tried to do
* # {{this.id}} {{lookup this.fields 'System.WorkItemType'}} - {{lookup this.fields 'Custom.Cliente'}}

but not  work

**Comment by rfennell** [comment]
Use the dump payload tools (see the [WIKI](https://github.com/rfennell/AzurePipelines/wiki)) to save out the payload passed into the Handlebars templates as a JSON file. 

You can then look in the WI > Fields. Chances are the name of the custom field is not what you are expecting.

**Reply by brux88** [reply]
if I enable the dump, I see it is called Custom.Cliente, but I can't get it to write in the release notes.

this is part of my dump file:
![2022-02-21_10-51-56](https://user-images.githubusercontent.com/3495339/154930774-351a151b-8051-4da8-ad07-308aef9765df.png)

**Comment by rfennell** [comment]
Strange, not seen that before. What do you see if use `{{JSON this.fields}}` inside your loop to dump out the WI field block?

**Reply by brux88** [reply]
I have to put it in place of the  {{lookup this.fields 'Custom.Cliente'}} ??

**Reply by rfennell** [reply]
Like this

```
{{#forEach this.workItems}}
{JSON this.fields}}
{{/forEach}}
```

The `JSON` function does a dump of any object  you pass it

**Reply by brux88** [reply]
perhaps a parenthesis is missing in your answer.

i tried  but show me this error:

2022-02-22T11:31:24.2628400Z ##[error]Error Processing handlebars [Error: Missing helper: "JSON"]


this is my template:
`# 🚀 Release {{buildDetails.buildNumber}}

**Release Number** : {{releaseDetails.name}}
**Release completed** : {{moment  releaseDetails.modifiedOn format="DD/MM/yyyy"}} 
**Build Number**: {{buildDetails.id}}

# Released Work Items
{{#forEach builds}}
{{#forEach this.workitems}}
* #{{this.id}} {{lookup this.fields ‘System.WorkItemType’}} - {{lookup this.fields ‘Custom.Cliente’}}
---
{{JSON this.fields}}
---
{{/forEach}}
{{/forEach}}


`

**Reply by rfennell** [reply]
Sorry, I wrote that answer from memory, it should be lower case as per the [WIKI ](https://github.com/rfennell/AzurePipelines/wiki/GenerateReleaseNotes---Node-based-Cross-Platform-Task#handlebar-extensions)

> In addition to the [Handlebars Helpers](https://github.com/helpers/handlebars-helpers) extension library, there are also some custom Helpers pre-loaded specific to the needs of this Azure DevOps task

> json that will dump the contents of any object. This is useful when working out what can be displayed in a template, though there are other ways to dump objects to files (see below)
> ## The contents of the build object
> {{json buildDetails}}

**Comment by brux88** [comment]
entering {{json this.fields}}, i show this filed:

,"Custom.Cliente":"ORTOMI "}


but if I insert in my model 
{{lookup this.fields'Custom.Cliente'}}


not show nothing
